classes/game.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../item'
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at
+
+  def initialize(player, last_played)
+    super(id, publish_date)
+    @multiplayer = player
+    @last_played_at = last_played
+  end
+
+  def can_be_archived?
+    (Date.today.year - @last_played_at.year) > 2
+  end
```

Comment:
- [ ] you should also call super here as you did for the book and the music album